bolt/lib/Rewrite/RewriteInstance.cpp

```diff
@@ -2439,6 +2439,21 @@ void RewriteInstance::readDynamicRelocations(const SectionRef &Section,
     if (Symbol)
       SymbolIndex[Symbol] = getRelocationSymbol(InputFile, Rel);
 
+    const uint64_t SymAddress = SymbolAddress + Addend;
```

Comment:
nit: `SymAddress` is very similar to `SymbolAddress`. How about `ReferencedAddress` for clarity?

Reply:
I've changed the name to `ReferencedAddress`, I've also changed `SymOffset` to `ReferencedOffset` for the same reason.